frontend/src/my-books/ShelfModal.css

```diff
@@ -0,0 +1,77 @@
+.shelf-modal-container {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  padding-top: 32px;
+  padding-bottom: 32px;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+    sans-serif;
+}
+
+.modal-title {
+  color: black;
+  font-size: 40pt;
+  letter-spacing: -0.5px;
+  font-weight: bold;
+}
+
+.shelf-modal-desc-container {
+  font-size: 16pt;
+  padding-top: 40px;
+  padding-bottom: 60px;
+}
+
+.PrivateHiddenCss-smDown-44 {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.shelf-modal-desc-container > .PrivateHiddenCss-smDown-44 > p {
+  width: 30%;
+  margin-right: 15px;
+}
+
+.shelfInput > .MuiOutlinedInput-root {
+  width: 100%;
+  margin-right: 600px;
+  border-radius: 20px;
+}
+
+.shelf-button-container {
+  display: flex;
+  justify-content: space-between;
+}
+.shelf-modal-button {
+  width: 100px;
+}
+
+@media screen and (max-width: 959px) {
+  .shelfInput > .MuiOutlinedInput-root {
+    width: 100%;
+    margin-right: 600px;
+    border-radius: 20px;
+  }
+  .PrivateHiddenCss-mdUp-46 {
```

Comment:
We can't guarantee that Material-UI will always assign 46 to the element being targeted here. Please create a unique class specific to this element. The same applies to other styles in this file.